azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/jobdetailspage.vm

```diff
@@ -37,6 +37,13 @@
     var jobId = "${jobid}";
     var attempt = ${attempt};
   </script>
+  <script type="text/javascript">
+    $(document).ready(function () {
+      #if ($jobFailed == "true")
```

Comment:
The velocity embedding would be the only advantage of velocity that I could think of

Reply:
Yes, it's convenient to use javascript within velocity template.